### PR TITLE
Sntlrnd 3786 script to manage call recordings using php

### PR DESCRIPTION
**Pull request by harshpatel1242**

### CODE REVIEW

File: sample-scripts/call-recording/Call_Recording.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+// base urls
+$base_url = "https://public-api.sonetel.com/";
+
+// function will change the format of date
+function date_format_change($date){
+    return str_replace(":", "%3A", $date);
+}
+
+//Enter Date
+$created_date_max = "ENTER THE MAX DATE";
```
Comment (aashish-joshi) [inline sample-scripts/call-recording/Call_Recording.php:11]: Perhaps it is better to use the date format we expect. For example, `$created_date_max = "yyyy-mm-ddThh:mm:ssZ"`.

This makes it clear to the customer what date format we're expecting.

**Review by aashish-joshi — CHANGES_REQUESTED** [review]
1. `Call_Forwarding.php` should be removed from the PR.
2. We may need to discuss if we should have the three call recording functions in different scripts to simplify things.
3. A few other changes needed in the script.

File: sample-scripts/call-recording/Call_Recording.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
+<?php
+// base urls
+$base_url = "https://public-api.sonetel.com/";
+
+// function will change the format of date
+function date_format_change($date){
```
[the remark this answers is not in the document]
Comment (harshpatel1242) [reply sample-scripts/call-recording/Call_Recording.php:6]: @aashish-joshi Resolved

File: sample-scripts/call-recording/Call_Recording.php [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    $response_download_call_recordings = curl_exec($curl_download_call_recordings);
+    $response_download_call_recordings = json_decode($response_download_call_recordings);
+    curl_close($curl_download_call_recordings);
+    echo ($response_download_call_recordings->{'response'}->{'file'}->{'file_access_details'}->{'url'});
+
+}
+
+if($choice==2){
+        // USER HAVE TO ENTER CALL_RECORDING_ID
+        $call_recording_id = "ENTER CALL_RECORDING_ID";
```
[the remark this answers is not in the document]
Comment (harshpatel1242) [reply sample-scripts/call-recording/Call_Recording.php:84]: @aashish-joshi Resolved

File: sample-scripts/call-recording/Call_Recording.php [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+$created_date_min = "ENTER THE MIN DATE";
+date_format_change($created_date_min);
+
+// API Access token
+$access_token = "ENTER_ACCESS_TOKEN";
+
+// Sonetel account ID
+$acc_id =  "ENTER_ACCOUNT_ID";
+
+// #########################Choices for user######################
```
[the remark this answers is not in the document]
Comment (harshpatel1242) [reply sample-scripts/call-recording/Call_Recording.php:23]: @aashish-joshi Resolved

File: sample-scripts/call-recording/Call_Recording.php [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+// ########################retrive all call recordings##################
+
+function retrive_call_recordings($base_url, $acc_id, $access_token, $created_date_min, $created_date_max){
+    $curl_retrive_call_recordings = curl_init();
+
+    curl_setopt_array($curl_retrive_call_recordings, array(
+    CURLOPT_URL => $base_url.'call-recording?account_id='.$acc_id.'&'.$created_date_min.'=&'.$created_date_max.'=&type=voice_call&fields=voice_call_details',
+    CURLOPT_RETURNTRANSFER => true,
+    CURLOPT_CUSTOMREQUEST => 'GET',
+    CURLOPT_POSTFIELDS => "<file contents here>",
```
[the remark this answers is not in the document]
Comment (harshpatel1242) [reply sample-scripts/call-recording/Call_Recording.php:40]: @aashish-joshi Resolved

File: sample-scripts/call-recording/Call_Recording.php (hunk 1 above)
Comment (harshpatel1242) [reply sample-scripts/call-recording/Call_Recording.php:11]: @aashish-joshi Resolved

**Pull request merged**